src/api/Feed.js

```diff
@@ -95,6 +103,83 @@ const getItemWithPendingReply = <T: { replies?: Array<Comment> }>(item: T, reply
     return { replies: [...replies, reply], ...rest };
 };
 
+const parseFileActivitiesResponseForFeed = (response?: { entries: FileActivity[] }) => {
+    if (!response || !response.entries || !response.entries.length) {
+        return { entries: [] };
+    }
+
+    const data = response.entries;
+    const parsedData = [];
+
+    data.map(item => {
```

Comment:
Can we use `const parsedData = data.map(item => {` to get value out of the map call?